examples/SLAM-camera/js/hit-test-location-xr8.ts

```diff
@@ -29,8 +29,8 @@ export class HitTestLocationXR8 extends Component {
     private _tmpWorldPosition = vec3.create();
 
     init() {
-        ARSession.onSessionStarted.add(this.onSessionStarted);
-        ARSession.onSessionEnded.add(this.onSessionEnded);
+        ARSession.getSessionForEngine(this.engine).onSessionStarted.add(this.onSessionStarted);
+        ARSession.getSessionForEngine(this.engine).onSessionEnded.add(this.onSessionEnded);
```

Comment:
Rename onSessionStarted => onSessionStart (and similar) to be consistent with Engine API (This is also consistent with DOM events)